tests/test_main.py

```diff
@@ -0,0 +1,47 @@
+# Tests main() function
+import subprocess
+from datetime import datetime, timedelta
+from unittest.mock import patch
+
+import pytest
+
+from cats import CATSOutput, main, schedule_at
+from cats.CI_api_interface import API_interfaces, InvalidLocationError
+from cats.forecast import CarbonIntensityAverageEstimate
+
+API = API_interfaces["carbonintensity.org.uk"]
+
+AT_OUTPUT = "%a %b %d %H:%M:%S %Y"
+now_start = (datetime.now() + timedelta(minutes=1)).replace(second=0)
+now_end = now_start + timedelta(minutes=5)
+
+OUTPUT = CATSOutput(
+    CarbonIntensityAverageEstimate(20, now_start, now_end),
+    CarbonIntensityAverageEstimate(20, now_start, now_end),
+    "OX1",
+    "GBR",
+)
+
+
+@pytest.mark.skip
+def test_schedule_at():
+    schedule_at(OUTPUT, ["ls"])
+    assert now_start.strftime(AT_OUTPUT) in subprocess.check_output(["atq"]).decode(
+        "utf-8"
+    )
```

Comment:
Is there a point to skipping this test? I guess it'll fail because the command is `atq` and not `at`

Reply:
Forgot to disable the skip! This is checking that the job is actually scheduled -- atq lists the at queue.

Reply:
at seems to be unavailable in the GitHub CI env, so will rework to use a mock